### ISSUE TITLE
Error when saving Launch Configuration

### ISSUE THREAD

**Issue by pervel** [warpdotdev/Warp#2076]
### Describe the bug

I try to save my first ever Launch Configuration and get the error message "An issue was encountered while saving".

### To Reproduce

1. Goto File -> Save New Launch Configuration
2. Press "Save Configuration"

### Expected behaviour

I expected no error message and I expected Launch Configuration to be saved to a file.

### Screenshots

![Screenshot 2022-11-10 at 04 13 54](https://user-images.githubusercontent.com/2744/200992679-f67020d9-96e8-4660-9abe-99a7d168ddf4.png)
![Screenshot 2022-11-10 at 04 14 06](https://user-images.githubusercontent.com/2744/200992682-c0206e7a-49a0-4566-8ff8-ffdccc9f541c.png)

### Operating System

MacOS

### OS Version

10.15.7

### Shell Version

GNU bash, version 5.2.9(1)-release (x86_64-apple-darwin19.6.0)

### Warp Version

v0.2022.11.01.08.03.stable_00

### Does this block you from using Warp daily?

No

**Comment by dannyneira** [comment]
Hi Pervel, I'm Danny, a Support Engineer at Warp. Thanks for letting us know! We're sorry about this. So I'm unable to reproduce this issue, does Warp have the "Files and Folders" permissions required to create files in Mac `System Settings > Privacy and Security`?

**Comment by pervel** [comment]
Yes, I had already given Warp "Full Disk Access" permissions manually (I believe that implies "Files and Folders" permission too). So that is apparently not the issue.

The error message is not very informative. Are there any logs that can give more information?

**Comment by pervel** [comment]
I found the problem. I had assumed that the name of the file was already prefilled in the save dialog. It very much looks that way as you can see on the screenshot. If I do type a filename, it saves without errors.

So may I suggest two changes? First, the error message should be much better. It should say "missing filename" or something like that. Second, there should be a default filename or the filename field should be left blank.

**Comment by dannyneira** [comment]
Thanks for bringing this to our attention, yeah seems like a UX issue to have the name be "pre-filled" like that.
